docs/index.asciidoc

```diff
@@ -255,6 +255,10 @@ The AWS Region
 Used to enable recovery after crash/abnormal termination.
 Temporary log files will be recovered and uploaded.
 
+NOTE: If you're using multiple S3 outputs than make sure to always set
+<<plugins-{type}s-{plugin}-temporary_directory,`temporary_directory`>> to a
+unique directory, otherwise the recovery mechanism won't work correctly.
```

Comment:
```suggestion
NOTE: If you're using multiple S3 outputs, always set
<<plugins-{type}s-{plugin}-temporary_directory>> to a
unique directory. Otherwise the recovery mechanism won't work correctly.
```

Reply:
Tip: You don't have to add link override text if you want the link to use the heading. That is, `<<plugins-{type}s-{plugin}-temporary_directory>> links to `temporary_directory` so no override is needed.